django/santropolFeast/member/models.py

```diff
@@ -209,3 +209,21 @@ class Meta:
         related_name='allergic_clients',
         blank=True
     )
+
+
+class Referencing (models.Model):
+
+    class Meta:
+        verbose_name_plural = _('referents')
+
+    referent = models.ForeignKey('member.Member',
+                                 verbose_name=_('member'))
+
+    client = models.ForeignKey('member.Client',
+                               verbose_name=_('client'))
+
+    referral_reason = models.TextField(
+            verbose_name=_("referral_reason"))
+
+    date = models.DateField(verbose_name=_("referral_date"),
```

Comment:
Yeah, or create a save function who check if date is Null and set the current date if is True.
I don't know if auto_now overwrite a create form with the date field